### PR TITLE
Profilepage

### PR DESCRIPTION
**Pull request by MickeMarten**
Created profilepage which shows user name and email.
Made functionality so user can toogle between views with buttons.
Tested with different users

Remade some responsiveness in footer.
Added small animation when database dosen't find any movies on userinput.
Made searchinput rounded.
Added label for searchinput.

Changed the Type Tmovie properties to lowercase to fix TS-error.
Changed name of type User in definitions.ts file because it clashed with the Auth.Api type User.
Changed it to DbUser.

Skärmklipp

Skärmklipp

### CODE REVIEW

File: kino-nextjs/app/components/navigation/SearchBar/SearchInput.tsx [hunk 1]
```diff
@@ -19,9 +19,17 @@ export default function SearchMovies() {
   }, 600);
   return (
     <>
+      <label
+        htmlFor='searchMovies'
+        className='flex flex-row gap-1 text-custom_yellow self-center md:self-start'
+      >
+        Sök på din Bihjofilm!
```
Comment (oskber) [inline kino-nextjs/app/components/navigation/SearchBar/SearchInput.tsx:26]: Nice job! Maybe change this to something like "Vilken film vill du se?" or something else so its not the same as the placeholder? :)

**Review by oskber — APPROVED** [review]
Good job, looks nice! Good work on getting the user data as well.

File: kino-nextjs/app/components/navigation/SearchBar/SearchInput.tsx (hunk 1 above)
Comment (MickeMarten) [reply kino-nextjs/app/components/navigation/SearchBar/SearchInput.tsx:26]: Absolut 😀

**Review by ANschoolacc — APPROVED** [review]
Looks good. Nice job!

**Pull request merged**